lib/dfu.js

```diff
@@ -108,12 +109,16 @@ var that = module.exports = {
 		];
 
 		that.checkBinaryAlignment("-D " + binaryPath);
-		return utilities.deferredSpawnProcess("dfu-util", args);
+		return utilities.deferredChildProcess(that.getCommand() + ' ' + args.join(' '));
```

Comment:
Using `deferredSpawnProcess` produce a `ENOENT` error ( there is a reference for this in the long commit message for #65 ).
I do not remember precisely why it worked creating a child process instead of spawning a new one.